### PR TITLE
Adds Software estimation in the AI era blog post

### PR DESCRIPTION
**Pull request by Potolski**

### CODE REVIEW

File: src/content/blog/software-estimation-with-ai.md [hunk 1]
```diff
@@ -0,0 +1,181 @@
...
+
+With AI assistance, the same developer can describe the endpoint requirements in natural language and receive a complete implementation in minutes. The AI generates not just the core functionality, but comprehensive error handling, input validation, and even unit tests that cover edge cases the developer might have overlooked.
+
+However, this acceleration isn't uniform across all types of work. While boilerplate code generation has become almost instantaneous, the time required for architectural decisions, stakeholder communication, and complex debugging remains largely unchanged. This creates a new estimation challenge: understanding which tasks benefit from AI acceleration and which remain human dependent.
+
+The productivity boost also varies a lot based on developer experience and the specific AI tools being used. A junior developer might see their productivity increase by 300% on routine tasks, while a senior developer working on complex system integration might see only marginal improvements.
+
+## Understanding AI Impact Categories
```
Comment (brunozimpel) [inline src/content/blog/software-estimation-with-ai.md:48]: I feel this section could be broken down into subsections for high, medium, and low AI-impact tasks to improve overall readability

File: src/content/blog/software-estimation-with-ai.md [hunk 2]
```diff
@@ -0,0 +1,181 @@
...
+The goal isn't perfect estimates, it's estimates that improve decision making while adapting to rapidly evolving AI capabilities.
+
+## Conclusion
+
+**tl;dr:**
+
+- **AI is revolutionizing estimation**: Traditional historical data needs updating as AI tools change development velocity
+- **Use AI aware three-point estimation**: Factor in AI Optimistic(AO), AI Realistic(AR), and Traditional/Tealistic Scenarios(TR)
+- **Categorize by AI impact**: High impact tasks (CRUD, testing) vs low impact tasks (architecture, debugging) need different approaches
+- **Account for experience levels**: Junior devs see much more gains, than seniors on suitable tasks
```
Comment (brunozimpel) [inline src/content/blog/software-estimation-with-ai.md:176]: I don't think we have a comma between "gains" and "than":

```suggestion
- **Account for experience levels**: Junior devs see much more gains than seniors on suitable tasks
```

**Comment by brunozimpel** [comment]
Overall, I **really** liked the article, haven't read any piece about the impact of AI on estimation, and I really liked the breakdown you did regarding high/mid/low AI-impact tasks, I'll be using that when talking to some stakeholders for sure. 

I left a few comments on specific points of the text, and have four more generic comments:

- 1) I'd make more use of **bold** and _italic_ across the article to emphasize concepts; this is something I particularly missed reading over such a content-heavy piece.
- 2) As it's a longer article, I'd add another image to break the text a bit and give the reader some breathing room. Similar to what you did on line 74. 
- 3) Extremely nitpicking here, but the structure of "_this isn't just about ... it's about ..._" feels really gpt-made. If you have the time, I'd consider rewriting them
- 4) Do run the article through Grammarly, I found some minor issues like 'writing' x 'writting', commas, and so on that would be good to iron out before we publish it
